misc/theme-cicada/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=		theme-cicada
+PLUGIN_VERSION=		0.2
```

Comment:
Let's start at 0.1 and clear the PLUGIN_REVISION :)

Reply:
Is done..

Note sure how to to tidy of the format in bootstrap-select.less. Whichever way I do it looks naff.